sci-electronics/kicad/kicad-4.0.2-r2.ebuild

```diff
@@ -0,0 +1,201 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+PYTHON_COMPAT=( python2_7 )
+
+WX_GTK_VER="3.0"
+
+inherit cmake-utils flag-o-matic gnome2-utils python-single-r1 vcs-snapshot wxwidgets versionator xdg
+
+DESCRIPTION="Electronic Schematic and PCB design tools."
+HOMEPAGE="http://www.kicad-pcb.org"
+LIBCONTEXT_COMMIT="3d92a1a50f4749b5a92131a957c9615473be85b4"
+
+SERIES=$(get_version_component_range 1-2)
+
+SRC_URI="https://launchpad.net/${PN}/${SERIES}/${PV}/+download/${P}.tar.xz
+	http://downloads.kicad-pcb.org/libraries/${PN}-footprints-${PV}.tar.gz
+	!minimal? ( https://github.com/KiCad/${PN}-library/archive/${PV}.tar.gz -> ${P}-library.tar.gz )
+	i18n? ( https://github.com/KiCad/${PN}-i18n/archive/${PV}.tar.gz -> ${P}-i18n.tar.gz )
+	https://github.com/twlostow/libcontext/archive/${LIBCONTEXT_COMMIT}.zip -> ${PN}-libcontext.zip"
+
+LICENSE="GPL-2 Boost-1.0"
```

Comment:
The license should be GPL-2+ GPL-3+:

```
kicad/kicad.cpp-13- * modify it under the terms of the GNU General Public License
kicad/kicad.cpp:14: * as published by the Free Software Foundation; either version 2
kicad/kicad.cpp-15- * of the License, or (at your option) any later version.
```

```
pcbnew/router/router_tool.cpp-8- * under the terms of the GNU General Public License as published by the
pcbnew/router/router_tool.cpp:9: * Free Software Foundation, either version 3 of the License, or (at your
pcbnew/router/router_tool.cpp-10- * option) any later version.
```

I'll fix it as part of my merge, no need to re-push